admin/horace_on.m.template

```diff
@@ -30,27 +41,36 @@ if ~isempty(sw_start)
     spinw_on(default_spinw_path);
 end
 
+if nargin > 1
+    independent_herbert = true;
+else
+    independent_herbert = false;
+end
 herbert_initated=~isempty(which('herbert_init.m'));
 % if Herbert is not initiated, try to init it.
 if ~herbert_initated
-    default_herbert_path = build_default_her_path(default_horace_path);
+    if independent_herbert
+        herbert_path = non_default_her_path;
+    else
+        herbert_path = build_default_her_path(horace_path);
+    end
     try
-        init_herbert(default_herbert_path);
+        init_herbert(herbert_path);
     catch ME
         disp('HORACE_ON:wrong_dependencies - cannot initiate Herbert');
         rethrow(ME);
     end
 else % reinitialize Herbert on where it is now.
-    her_path = fileparts(which('herbert_init.m'));
-    init_herbert(her_path);
+    if independent_herbert
+        init_herbert(non_default_her_path);
```

Comment:
Why not move the `if independent_herbert` block outside the `if ~herbert_initiated` block and use `herbert_path` set there?

Reply:
because `independent_herbert` is used in both Herbert branches.